### DISCUSSION TITLE
Wifi  selection button

### DISCUSSION THREAD

**Discussion by ttanya** [SafeExamBrowser/seb-win-refactoring#133] [Q&A]
Hi

I'm using version 3.1.1 on Windows. Lately noticed that after a while of using SEB I have only one WIFI network available. In the first few minutes of using SEB all the  networks are available and I can chose any of them.
What could be the issue?
Thank you.

**Comment by dbuechel** [comment]
That most likely is due to the signal strength of the other networks. Wireless networks aren't as reliable as wired networks and there often can be significant fluctuation in signal strength, up to the point that a network is available in one moment and in the next moment it can't be reached anymore. Did you verify that the behavior is the same in Windows itself?

**Reply by ttanya** [reply]
`No, the networks are fine. They don't disappear at Windows without SEB.`

**Reply by dbuechel** [reply]
Could you please provide the application log files, to be found under `%LocalAppData%\SafeExamBrowser\Logs`?
